tests/bool.rs

```diff
@@ -0,0 +1,24 @@
+use serde_bin_prot::to_writer;
+
+mod common;
+use common::TestCase;
+
+fn bool_test_cases() -> Vec<TestCase<bool>> {
+    vec![TestCase::new(true, vec![1]), TestCase::new(false, vec![0])]
```

Comment:
Kinda nit but I think it makes it more clear that this is a byte vec
```suggestion
    vec![TestCase::new(true, vec![0x01]), TestCase::new(false, vec![0x00])]
```

Reply:
updated